### PR TITLE
Support LinuxParameters option with ECS

### PR DESCRIPTION
**Pull request by ayemos**
ECS now supports new LinuxParameters option where we can pass `--cap-add` and `--cap-drop` option to `docker run` .

@eagletmt please review 🙏

### CODE REVIEW

**Comment by eagletmt** [comment]
Specify `aws-sdk-ecs >= 1.2.0`.
https://github.com/aws/aws-sdk-ruby/commit/a801048aed60e203108c0a02c98f386ab5da2516#diff-74e9d9021188ae4f07ac370695885c9b

File: lib/hako/container.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ def initialize(app, definition, dry_run:)
       command
       user
       privileged
+      linux_parameters
```
Comment (eagletmt) [inline lib/hako/container.rb]: I expect all Hash values returned from accessor has symbolized keys.
Please define custom accessor like `log_configuration`.

**Comment by ayemos** [comment]
@eagletmt  please review again 🙏

**Comment by ayemos** [comment]
@eagletmt 👀

**Pull request merged**